demo/site/src/common/blocks/SpaceBlock.tsx

```diff
@@ -1,15 +1,10 @@
 "use client";
 import { type PropsWithData, withPreview } from "@comet/site-nextjs";
 import { type SpaceBlockData } from "@src/blocks.generated";
-import styled from "styled-components";
 
 export const SpaceBlock = withPreview(
     ({ data: { spacing } }: PropsWithData<SpaceBlockData>) => {
-        return <Root $spacing={spacing} />;
+        return <div style={{ height: `var(--spacing-${spacing})` }} />;
```

Comment:
I'm not 100% sure if this is good this way. (It's definitely the shortest)
The other option is to define for every spacing variant a class
`.d300 {
     height: var(--spacing-d300);
}`
and use this this way:
`<div className={styles[spacing]}>`

Reply:
I like this solution because it works without a mapping.